### PR TITLE
pantalaimon: 0.4 -> 0.5.1

### PR DESCRIPTION
**Pull request by symphorien**
###### Motivation for this change

update pantalaimon

###### Things done

updated nio as a new version is required by pantalaimon.

Tested that pantalaimon is functional.

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"` <- nix-review OOMs on my machine.
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by Valodim — APPROVED** [review]
lgtm, compiles and runs ok

File: pkgs/applications/networking/instant-messengers/pantalaimon/default.nix [hunk 1]
```diff
@@ -9,13 +9,16 @@
...
-  version = "0.4";
+  version = "0.5.1";
 
   disabled = pythonOlder "3.6";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1canj9w72wh1rcw6fivrvaahpxy13gb6gh1k8nss6bgixalvnq9m";
+  # pypi tarball miss tests
+  src = fetchFromGitHub {
```
Comment (Valodim) [inline pkgs/applications/networking/instant-messengers/pantalaimon/default.nix]: out of curiosity, is there a reason to use github over pypi?

Comment (emilazy) [reply pkgs/applications/networking/instant-messengers/pantalaimon/default.nix]: presumably because `pypi tarball miss tests`?

Comment (symphorien) [reply pkgs/applications/networking/instant-messengers/pantalaimon/default.nix]: yes.

Comment (Valodim) [reply pkgs/applications/networking/instant-messengers/pantalaimon/default.nix]: heh, of course. derp :)

**Pull request merged**